simpleclient_pushgateway/src/main/java/io/prometheus/client/exporter/PushGateway.java

```diff
@@ -268,9 +268,10 @@ void doRequest(CollectorRegistry registry, String job, Map<String, String> group
         writer.close();
       }
 
-      int response = connection.getResponseCode();
-      if (response != HttpURLConnection.HTTP_ACCEPTED) {
-        throw new IOException("Response code from " + url + " was " + response);
+      int responseCode = connection.getResponseCode();
+      if (responseCode != HttpURLConnection.HTTP_ACCEPTED) {
+        throw new IOException("Response code from " + url + " was " + responseCode
+            + ", message: " + connection.getResponseMessage());
```

Comment:
This will just print the standardized response message (like "Not Found" for 404 or "Internal Server Error" for 500). The important information, however, is in the body. (getContent() or something?)